Program.fs

```diff
@@ -355,51 +371,39 @@ let executeResolutionActions (cwd: string) (manager: NodeManager) (actions: Reso
         | _ -> None)
 
     if not (List.isEmpty uninstallPackages) then
+        // then there some packages we need to uninstall first
         let program, args =
             match manager with
-            | NodeManager.Npm ->
-                if Environment.isWindows
-                then "cmd", List.concat [ ["/C"; "npm"; "uninstall" ]; uninstallPackages; [ "--save" ] ]
-                else "npm", List.concat [ [ "uninstall" ]; uninstallPackages; ["--save" ]]
-            | NodeManager.Yarn ->
-                if Environment.isWindows
-                then "cmd", List.append [ "/C"; "yarn"; "remove" ] uninstallPackages
-                else "yarn", List.append [ "remove" ] uninstallPackages
+            | NodeManager.Npm -> "npm", [ yield "uninstall"; yield! uninstallPackages; yield "--save" ]
+            | NodeManager.Yarn -> "yarn", [ yield "remove"; yield! uninstallPackages ]
 
         logger.Information("Uninstalling [{Libraries}]", String.concat ", " uninstallPackages)
-        CreateProcess.fromRawCommand program args
+        CreateProcess.xplatCommand program args
         |> CreateProcess.withWorkingDirectory cwd
         |> CreateProcess.ensureExitCodeWithMessage (sprintf "Error while uninstalling [%s]" (String.concat ", " uninstallPackages))
         |> CreateProcess.redirectOutput
         |> Proc.run
         |> ignore
 
-
     if not (List.isEmpty installPackages) then
         let packagesToInstall =
             installPackages
             |> List.map (fun (package, version) -> sprintf "%s@%s" package version)
 
         let program, args =
             match manager with
-            | NodeManager.Npm ->
-                if Environment.isWindows
-                then "cmd", List.concat [ ["/C"; "npm"; "install"]; packagesToInstall; ["--save"] ]
-                else "npm", List.concat [ ["install" ] ; packagesToInstall; [ "--save" ] ]
-            | NodeManager.Yarn ->
-                if Environment.isWindows
-                then "cmd", List.concat [ [ "/C"; "yarn"; "add"; ]; packagesToInstall ]
-                else "yarn", List.concat [ [ "add" ]; packagesToInstall ]
-
-        logger.Information("Installing [{Libraryies}]", packagesToInstall |> String.concat ", ")
-        CreateProcess.fromRawCommand program args
+            | NodeManager.Npm -> "npm", List.concat [ ["install" ]; packagesToInstall; [ "--save" ] ]
```

Comment:
Can we use `yield` and `yield!` syntax here too?

I personnally find it easier to read (when having a lot of arguments we can even place each element of the list on a new line. This is also make the code consistent :)

Reply:
Done! 

I was debating it myself whether I should use `List.concat` or `yield` but the latter reads better